api/server.js

```diff
@@ -1,5 +1,7 @@
 const app = require("./app");
+const cron = require('./cron-jobs/run-cron-jobs')
```

Comment:
Tiny thing, but this and line 5 should have a semi-colon at the end.

(At some point we should switch the API to Typescript and could enforce the same coding style as the front-end app e.g. no semi-colons for line endings)